helm/defectdojo/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "2.44.2"
 description: A Helm chart for Kubernetes to install DefectDojo
 name: defectdojo
-version: 1.6.178
+version: 1.6.179
```

```suggestion
version: 1.6.178
```
Bumps here are performed during a release - reverting this